src/Components/DrillAnimation.js

```diff
@@ -1,29 +1,245 @@
-import React, { Component } from 'react';
-import { StyleSheet, View, Text, Dimensions } from 'react-native';
-import Animation from './Animation';
+import React, { useState } from 'react';
+import { ScrollView, StyleSheet, View, Text, Dimensions, TouchableOpacity } from 'react-native';
 import { WebView } from 'react-native-webview';
+import { LinearGradient } from 'expo-linear-gradient';
+
+import Animation from './Animation';
+import VimeoVideo from './VimeoVideo';
+import theme from '../styles/theme.style';
+
+const DrillAnimation = props => {
+  const [count, setCount] = useState(0);
+
+  const incrementCount = () => {
+    setCount((count + 1) % (props.drill.steps.length + 1));
+  };
+
+  const displayNextStep = () => {
+    if (count + 1 === props.drill.steps.length) {
+      return (
+        <View style={styles.infoWrapper}>
+          <View style={styles.description}>
+            <View style={styles.wrapperFinish}>
+              <Text style={styles.fitnessNext}>Finish</Text>
+            </View>
+          </View>
+          <View style={styles.lines} />
+        </View>
+      );
+    } else {
+      return (
+        <View style={styles.infoWrapper}>
+          <View style={styles.description}>
+            <View style={styles.subWrapper}>
+              <Text style={styles.fitnessNext}>{props.drill.steps[count + 1].count}</Text>
```

Comment:
Maybe it would be easier to understand if count was called `currentStepIndex` ?

Reply:
Done